src/popup/App.vue

```diff
@@ -216,6 +235,16 @@ export default defineComponent({
 
       watchConnectionStatus();
 
+      if (IS_EXTENSION) {
+        transferSendData.value = getLocalStorageItem(
+          [TRANSFER_SEND_DATA_LOCAL_STORAGE_KEY],
+        );
+        if (!transferSendData.value) {
+          unwatch();
+        }
+        removeLocalStorageItem([TRANSFER_SEND_DATA_LOCAL_STORAGE_KEY]);
+      }
```

Comment:
My previous comment applies also to this block of code.
In my opinion there is no need to put this into `onMounted` hook.